api.planx.uk/modules/auth/controller.ts

```diff
@@ -39,7 +39,8 @@ function setJWTCookie(returnTo: string, res: Response, req: Request) {
     maxAge: new Date(
       new Date().setFullYear(new Date().getFullYear() + 1),
     ).getTime(),
-    sameSite: "none",
+    // the JWT/auth cookies should be sent only between the API server and editor
+    sameSite: "strict",
```

Comment:
Please correct me if this doesn't make sense!

Reply:
I think this needs to be "none" for login to work between the Pizzas and Staging API. Worth rebasing this once https://github.com/theopensystemslab/planx-new/pull/3890 is merged and testing?

Reply:
Riiight, that's the motivation for `none`, I hadn't thought about that case. Although this change should affect both means of authenticating, and Google seems to work on the [pizza](https://3803.planx.pizza/) 🤔

Reply:
Will do as you suggest and see what happens